### ISSUE TITLE
Incoming Request Logging

### ISSUE THREAD

**Issue by arb** [hapijs/good#359]
Add something to listen for incoming requests that isn't an extension method. At present, there isn't a great way to get an "only once" kind of event from hapi because "request-internal" happens multiple times.

**Comment by davidbenko9** [comment]
+1

**Comment by MichalZalecki** [comment]
Logging requests was working well in 5.x now I cannot get any usefull info.

**Comment by arb** [comment]
It's hard to give you any help or advice without seeing your configuration and listing of versions you are using. If you're having problems, please open a new issue.

**Comment by MichalZalecki** [comment]
@arb Sure. 

This is example log from one of my apps

```
160518/205444.990, [response], http://localhost:3000: post /graphql {} 200 (99ms)
```

```
"good": "^6.6.0",
"good-console": "^5.3.1",
"hapi": "^13.2.2",
```

```
{
  register: require("good"),
  options:  {
    reporters: [{
      reporter: require("good-console"),
      events: { log: "*", response: "*" },
    }],
  },
},
```

I would like to be able to get same/similar log with

```
"good": "^7.0.0",
"good-console": "^6.1.1",
"hapi": "^13.3.0",
```

```
{
  register: require("good"),
  options: {
    ops: {
      interval: 1000,
    },
    reporters: {
      console: [{
        module: "good-console",
        args: [{ log: "*", response: "*" }],
      }],
    },
  },
},
```

No luck though. Sorry for not being more explicit.

Thanks!

**Comment by arb** [comment]
Please be sure to read the documentation and look at the [examples](https://github.com/hapijs/good/blob/master/examples/log-to-console.md). Your `console` reporter pipeline doesn't end with a write stream. That's why you aren't seeing anything. You want to add `"stdout"`.

**Comment by rajeshwarpatlolla** [comment]
I have the same problem.

My options are 

`  const goodOptions = {
    ops: {
      interval: 1000,
    },
    reporters: {
      console: [
        {
          module: '@hapi/good-squeeze',
          name: 'Squeeze',
          args: [{ log: '*', response: '*' }],
        },
        {
          module: '@hapi/good-console',
        },
        'stdout',
      ],
    },
  };`

`
 await server.register([
    Inert,
    Vision,
    {
      plugin: HapiSwagger,
      options: swaggerOptions,
    },
    {
      plugin: require('@hapi/good'),
      goodOptions,
    },
  ]);
`

When i run my server, server is starting and the apis are working fine. But i don't see any logs.

`
"@hapi/good": "^8.2.2",
    "@hapi/good-console": "^8.1.2",
    "@hapi/good-squeeze": "^5.2.1",
    "@hapi/hapi": "^18.3.2",
    "@hapi/inert": "^5.2.1",
    "@hapi/joi": "^15.1.1",
    "@hapi/vision": "^5.5.3",

node -v
v10.16.0

npm -v
6.9.0`

Any help would be really appreciated.

**Comment by hueniverse** [comment]
@rajeshwarpatlolla commenting on a closed issue that's over 3 years old is not the way to get help...I would suggest trying the Slack channel. You should also write a super simple server with the bare minimum and try to get it working. Alternatively, try one of the examples.

**Comment by rajeshwarpatlolla** [comment]
Yeah, it's old. I have already raised a new issue. My code base just has basic server setup, nothing much. I tried the code from examples itself.

**Comment by hueniverse** [comment]
Next time delete duplicate comments. Don't spam.
